types/next-redux-saga/package.json

```
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
+        "@types/next-redux-wrapper": "^2.0.2",
```

Since `@types/next-redux-wrapper` will be deprecated, this needs the original package.

```suggestion
        "next-redux-wrapper": "^3.0.0",
```